cmd/tegola/cmd/cache.go

```diff
@@ -315,14 +315,19 @@ func sendTiles(zooms []uint, c chan *slippy.Tile) error {
 			}
 			tile := slippy.NewTile(z, x, y, 0, tegola.WebMercator)
 
+			c <- tile
+
 			// range
 			for _, zoom := range zooms {
```

Comment:
Why do you need it? You have all the information in the file `cacheFile`.

Reply:
As I said above, one of the use cases ((imposm3)[https://github.com/omniscale/imposm3]) gives a file with uniform zooms. In such a case, the zoom supplied in the file is not really important.

The line you commented ensures the behavior of seeding the tiles directly from the file. This will have to be redesigned...